langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -206,6 +275,16 @@ public isolated function getTarget(ProcessingInstruction x) returns string = @ja
 
 # Returns the content of a processing instruction or comment item.
 #
+# ```ballerina
+# // Get the content of an XML Processing Instruction
+# xml:ProcessingInstruction person = xml `<?person name?>`;
+# string content = person.getContent();
+# 
+# // Get the content of an XML comment
```

Comment:
I think we can remove the comments

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/f0dac9d10c92de79bf074968b42be066e0b45d1c